cluster-provision/centos9/Dockerfile

```diff
@@ -27,11 +27,7 @@ ENV CENTOS_URL https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Str
 
 RUN /download_box.sh ${CENTOS_URL}
 
-# Pin to earlier versions of vmlinuz & initrd.img due to issue with latest
```

Comment:
I think we should include this information inside some troubleshooting document, including a reference to the commit, so that we can apply this again if required.

This could also be done for other workarounds we had to do.